Add CloseGroup/AbortGroup to mappings (#1008)
ietf-tapswg/api-drafts: draft-ietf-taps-impl.md

Abort:
: Calling `Abort` on a TCP Connection indicates that the Connection should be immediately closed by sending a RST to the peer (ABORT.TCP).

> CloseGroup:
> : Calling `CloseGroup` on a TCP Connection (CLOSE.TCP) is identical to calling `Close`.
> 
> AbortGroup:
> : Calling `AbortGroup` on a TCP Connection (ABORT.TCP) is identical to calling `Abort`.
> 

Author: ```suggestion
CloseGroup:
: Calling `CloseGroup` on a TCP Connection (CLOSE.TCP) is identical to calling `Close` on all TCP Connections
within a group.

AbortGroup:
: Calling `AbortGroup` on a TCP Connection (ABORT.TCP) is identical to calling `Abort` on all TCP Connections
within a group.

```

Abort:
: Calling `Abort` on a TCP Connection indicates that the Connection should be immediately closed by sending a RST to the peer (ABORT.TCP).

CloseGroup:

> : Calling `CloseGroup` on a TCP Connection (CLOSE.TCP) is identical to calling `Close` on all TCP Connections
> within a group.

Reviewer: How about:

```suggestion
: Calling `CloseGroup` on a TCP Connection (CLOSE.TCP) is identical to calling `Close` on this Connection and on all Connections in the same ConnectionGroup.
```
This is essentially what it says in Section 10 in the Interface draft, and perhaps the clearest way to phrase this.

Reviewer 2: I like this - I think it should be phrased using that style for all the calls below. "within a group" is just a touch too vague IMO... it needs to be the group this connection is in, even though this is super obvious.

Author: Applied, thanks!

> AbortGroup:
> : Calling `AbortGroup` on a TCP Connection (ABORT.TCP) is identical to calling `Abort` on all TCP Connections
> within a group.

Author: ```suggestion
AbortGroup:
: Calling `AbortGroup` on a TCP Connection (ABORT.TCP) is identical to calling `Abort` on this Connection and on all Connections in the same ConnectionGroup.
```

Abort:
: Calling `Abort` on a UDP Connection (ABORT.UDP(-Lite)) is identical to calling `Close`.

> CloseGroup:
> : Calling `CloseGroup` on a UDP Connection (ABORT.UDP(-Lite)) is identical to calling `Close`.
> 
> AbortGroup:
> : Calling `AbortGroup` on a UDP Connection (ABORT.UDP(-Lite)) is identical to calling `Close`.
> 

Author: ```suggestion
CloseGroup:
: Calling `CloseGroup` on a UDP Connection (ABORT.UDP(-Lite)) is identical to calling `Close` on all
UDP Connections within a group.

AbortGroup:
: Calling `AbortGroup` on a UDP Connection (ABORT.UDP(-Lite)) is identical to calling `Close` on all
UDP Connections within a group.

```

Abort:
: Calling `Abort` on a UDP Connection (ABORT.UDP(-Lite)) is identical to calling `Close`.

> CloseGroup:
> : Calling `CloseGroup` on a UDP Connection (ABORT.UDP(-Lite)) is identical to calling `Close` on all
> UDP Connections within a group.
> 
> AbortGroup:
> : Calling `AbortGroup` on a UDP Connection (ABORT.UDP(-Lite)) is identical to calling `Close` on all
> UDP Connections within a group.
> 

Author: ```suggestion
CloseGroup:
: Calling `CloseGroup` on a UDP Connection (ABORT.UDP(-Lite)) is identical to calling `Close` on this Connection and on all Connections in the same ConnectionGroup.

AbortGroup:
: Calling `AbortGroup` on a UDP Connection (ABORT.UDP(-Lite)) is identical to calling `Close` on this Connection and on all Connections in the same ConnectionGroup.
```

Abort:
: Calling `Abort` on a UDP Multicast Receive Connection (ABORT.UDP(-Lite)) is identical to calling `Close`.

CloseGroup:

> : Calling `CloseGroup` on a UDP Multicast Receive Connection (ABORT.UDP(-Lite)) is identical to calling `Close`.

Author: ```suggestion
: Calling `CloseGroup` on a UDP Multicast Receive Connection (ABORT.UDP(-Lite)) is identical to calling `Close` on all UDP Multicast Receive  Connections within a group.
```

AbortGroup:

> : Calling `AbortGroup` on a UDP Multicast Receive Connection (ABORT.UDP(-Lite)) is identical to calling `Close`.

Author: ```suggestion
: Calling `AbortGroup` on a UDP Multicast Receive Connection (ABORT.UDP(-Lite)) is identical to calling `Close`
on all UDP Multicast Receive  Connections within a group.
```

Abort:
: Calling `Abort` on a UDP Multicast Receive Connection (ABORT.UDP(-Lite)) is identical to calling `Close`.

> CloseGroup:
> : Calling `CloseGroup` on a UDP Multicast Receive Connection (ABORT.UDP(-Lite)) is identical to calling `Close` on all UDP Multicast Receive  Connections within a group.
> 
> AbortGroup:
> : Calling `AbortGroup` on a UDP Multicast Receive Connection (ABORT.UDP(-Lite)) is identical to calling `Close`
> on all UDP Multicast Receive  Connections within a group.
> 

Author: ```suggestion
CloseGroup:
: Calling `CloseGroup` on a UDP Multicast Receive Connection (ABORT.UDP(-Lite)) is identical to calling `Close` on this Connection and on all Connections in the same ConnectionGroup.

AbortGroup:
: Calling `AbortGroup` on a UDP Multicast Receive Connection (ABORT.UDP(-Lite)) is identical to calling `Close`
on this Connection and on all Connections in the same ConnectionGroup.

```
